tests/csm/rest/test_csm_load.py

```diff
@@ -118,6 +165,161 @@ def test_poc(self):
+            rampup=test_cfg["rampup"],
+            loop=test_cfg["loop"])
+        assert result, "Errors reported in the Jmeter execution"
+
+        stop_res = run_data_chk_obj.stop_io_async(users=data, di_check=True, eventual_stop=True)
+        self.log.info("stop_res -  %s", stop_res)
+
+        self.log.info("Perform & Verify GET API to get capacity usage stats")
+        resp = self.csm_obj.get_user_capacity_usage("user", username)
+        assert resp.status_code == HTTPStatus.OK, \
+                "Status code check failed for get capacity"
+        t_obj = resp.json()["capacity"]["s3"]["users"][0]["objects"]
+        t_size = resp.json()["capacity"]["s3"]["users"][0]["used"]
+        m_size = resp.json()["capacity"]["s3"]["users"][0]["used_rounded"]
+        self.log.info("objects -  %s", t_obj)
+        self.log.info("used capacity-  %s", t_size)
+        self.log.info("used_rounded capacity-  %s", m_size)
+        assert t_obj > 0, "Number of objects is Zero"
+        assert t_size > 0, "Used Size is Zero"
+        assert m_size > 0, "Total Size is Zero"
+        assert m_size >= t_size, "Used - Used Rounded Size mismatch found"
+
+        self.log.info("##### Test completed -  %s #####", test_case_name)
+
+
+    # pylint: disable=too-many-statements
+    @pytest.mark.skip(reason="not_in_main_build_yet")
+    @pytest.mark.jmeter
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.tags('TEST-44794')
+    def test_44794(self):
+        """
+        CSM load testing in degraded mode to view Degraded capacity
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.test_cfgs["test_44794"]
+        self.log.info("Step 1: Create accounts for I/O")
+        mgm_ops = ManagementOPs()
+        users = mgm_ops.create_account_users(nusers=test_cfg["users_count"], use_cortx_cli=False)
+        data = mgm_ops.create_buckets(nbuckets=test_cfg["buckets_count"], users=users)
+
+        self.log.info("Step 2: [Start] Shutdown the data pod safely")
+        num_replica = 0
+        self.log.info("Get pod to be deleted")
+        sts_dict = self.csm_obj.master.get_sts_pods(pod_prefix=const.POD_NAME_PREFIX)
+        sts_list = list(sts_dict.keys())
+        self.log.debug("%s Statefulset: %s", const.POD_NAME_PREFIX, sts_list)
+        sts = self.system_random.sample(sts_list, 1)[0]
+        delete_pod = sts_dict[sts][-1]
+        self.log.info("Pod to be deleted is %s", delete_pod)
+        set_type, set_name = self.csm_obj.master.get_set_type_name(pod_name=delete_pod)
+        if set_type == const.STATEFULSET:
+            resp = self.csm_obj.master.get_num_replicas(set_type, set_name)
+            assert resp[0], resp
+            self.num_replica = int(resp[1])
+            num_replica = self.num_replica - 1
+
+        self.log.info("Shutdown random data pod by making replicas=0 and "
```

Comment:
we are not making replicas=0 with stafefulset, so u can just say with replica method

Reply:
updated